.gitignore

```diff
@@ -7,4 +7,8 @@ docker-compose.yml
 # Rust
 /target
 
+# Zig
```

Comment:
Please create a `.gitignore` in the zig directory. Similar to 

https://github.com/sharkbench/sharkbench/blob/main/benchmark/computation/dart/.gitignore